code/src/Components/AlbumsContainer.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Album } from './Album';
+import './albumContainer.css'
+
+export const AlbumsContainer = ({ albums }) => {
+  return (
+    <article className="album-container">
+      {albums.map((album) => {
+        return (
+          // eslint-disable-next-line max-len
+          <Album key={album.id} link={album.external_urls.spotify} title={album.name} artists={album.artists} cover={album.images[1].url} />
```

Comment:
Idea how to fix long code
```suggestion
<Album 
  key={album.id} 
  link={album.external_urls.spotify} 
  title={album.name} 
  artists={album.artists} 
  cover={album.images[1].url} 
/>
```

Reply:
Super, big thanks! :)